src/pages/Home.tsx

```diff
@@ -1,60 +1,82 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
please, remove unused declaration for `React`, as it breaks the deployment on Vercel :)

Reply:
thanks for review :)